### PR TITLE
Add support for svg external content

### PR DESCRIPTION
**Pull request by Neatoro**

### CODE REVIEW

File: feature-detects/svg/externalresources.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+}
+!*/
+/* DOC
+Detects support for External Resources in SVG.
+
+See [this discussion](https://github.com/Modernizr/Modernizr/issues/1409) regarding SVG external content.
+*/
+define(['Modernizr'], function(Modernizr) {
+  Modernizr.addTest('externalresources', function() {
+    return document.implementation.hasFeature('http://www.w3.org/TR/SVG11/feature#ExternalResourcesRequired', '1.1');
```
Comment (rejas) [inline feature-detects/svg/externalresources.js]: Doesnt that violate  https://github.com/Modernizr/Modernizr/issues/1409#issuecomment-51089302 : 
"unfortunately, modernizr can't really use any external uris, ever. all of our tests need to be self contained and not realiant upon a network connection."

**Comment by patrickkettner** [comment]
No, this is something that is built into browsers. If you Google hasImplementation you'll find a bunch about this

**Comment by rejas** [comment]
I see... https://developer.mozilla.org/en-US/docs/Web/API/DOMImplementation/hasFeature says it always returns true in modern browser since its deprecated
That shouldnt be a problem for this issue since modern browsers seem to support the svg feature anyways.
So @Neatoro would you be willing to update your PR agaisnt the latest master sources to get this merged?

**Comment by patrickkettner** [comment]
@rejas I missed the original context of this in my last reply (was going off of the email). 

This should closed, not merged. It is not actually detecting anything (as that function always returns true)

**Pull request closed without merging**